openra/tests/test_commands.py

```diff
@@ -71,6 +72,7 @@ def test_it_creates_a_super_user_with_the_details_provided(self):
         )
 
     def test_it_imports_the_sample_maps(self):
+        return
```

Comment:
These two `return`s seem dangerous without even a TODO.

Reply:
Yes I should have taken them out before I pushed the changes up. The issue is, these tests actually run utility commands as they can't actually be mocked safely yet, so they take maybe 30 seconds or more to run, when all of the other tests run in less than a second.

I'll get them re-enabled though as they shouldn't have been committed like this.

Reply:
Ok these are now enabled.